main/inc/lib/CoursesAndSessionsCatalog.class.php

```diff
@@ -1520,7 +1630,8 @@ public static function getCatalogPagination(
         $pageTotal,
         $categoryCode = '',
         $action = '',
-        $fields = []
+        $fields = [],
+        $sortKeys
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Arguments with default values must be at the end of the argument list](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5354773)